bugsnag/django/__init__.py

```diff
@@ -22,9 +22,8 @@ def add_django_request_to_notification(notification):
 
     if hasattr(request, 'user') and request.user.is_authenticated():
         try:
-            name = " ".join([request.user.first_name or '',
-                             request.user.last_name or ''])
-            notification.set_user(id=request.user.username,
+            name = request.user.get_full_name()
+            notification.set_user(id=request.user.get_username(),
                                   email=request.user.email, name=name)
```

Comment:
`request.user.email` might also not exist, or it might not be a string.

Example that I might build a custom user object that makes the email a foreign relation to an `Email` model.

See [Specifying a custom User model](https://docs.djangoproject.com/en/1.9/topics/auth/customizing/#specifying-a-custom-user-model)